src/redux/actions/concepts/dictionaryConcepts.js

```diff
@@ -125,13 +128,19 @@ export const fetchDictionaryConcepts = (
   const filterByClass = getState().concepts.filteredByClass;
 
   if (filterBySource.length > 0 && filterByClass.length > 0) {
-    url = `${conceptType}/${conceptOwner}/sources/${conceptName}/concepts/?q=${query}&limit=${limit}&page=${page}&verbose=true&source=${filterBySource.join(',')}&conceptClass=${filterByClass.join(',')}`;
+    url = `${conceptType}/${conceptOwner}/collections/${conceptName}/concepts/?q=${query}&limit=${limit}&page=${page}&verbose=true&source=${filterBySource.join(
```

Comment:
Is this change part of the ticket?

Reply:
@Duncanian No. Let me undo such unrelated changes. Thanks for this.